src/Configuration.php

```diff
@@ -279,4 +288,27 @@ public function hideResources(array $resources = []): static
 
         return $this;
     }
+
+    /**
+     * @param  class-string<NavLinkRenderer>  $rendererClass
+     * @return $this
+     */
+    public function navRenderer(string $rendererClass): static
+    {
+        $this->navRenderers[$rendererClass::$renders] = $rendererClass;
+
+        return $this;
+    }
+
+    public function getNavRenderers(): array
```

Comment:
ok, first note: we cant use the plugin configuration class for any FE stuff. since the panel is not available, and `Filament` will load the default one. and in some cases the plugin is not registered in the default panel.

so if I have `AdminPanel` and `CmsPanel` and sky only registered in `CmsPanel` but the `->default()` is `CmsPanel`... :) will it will throw an exception.

that is way I kept the config file, for all frontend stugg. just need to move this one to the config file.

Reply:
ohh - fair call out. I hadn't gotten to my task of "customize the zeus theme/template" yet! I've only been testing the navigation in my own sites templates so far. So I think once I setup the Blog's theme to match my template it'll make more sense to me what's going on here.